### ISSUE TITLE
Unable to load a podspec from `SQLite.swift.podspec` on `swift-4` branch

### ISSUE THREAD

**Issue by t4ec** [stephencelis/SQLite.swift#737]
Hello,

I am trying to install SQLite.swift from swift-4 branch which worked just fine few weeks ago. But now when I use following Podfile:
```
use_frameworks!
platform :ios, '11'

# ...

target "Example" do
  pod 'SQLite.swift', :git => 'https://github.com/stephencelis/SQLite.swift.git', :branch => 'swift-4'
end
```
I end up with error:
```
Pre-downloading: `SQLite.swift` from `https://github.com/stephencelis/SQLite.swift.git`, branch `swift-4`
[!] Unable to find a specification for 'SQLite.swift'.

[!] Unable to load a podspec from `SQLite.swift.podspec`, skipping:

Pod::DSLError
```

Master branch works just fine. Could you please let me know if there is any misconfiguration on my side or it is a bug. 

Thank you in advance!

**Comment by fdematos** [comment]
Same here

**Comment by jberkel** [comment]
Can't reproduce. Try deleting the cache: 

```sh
$ rm -rf ~/Library/Caches/CocoaPods/Pods/External/SQLite.swift
```

**Comment by t4ec** [comment]
> Can't reproduce. Try deleting the cache:
> 
> `$ rm -rf ~/Library/Caches/CocoaPods/Pods/External/SQLite.swift`

Sorry, that didn't help.

BTW recently I have updated to High Sierra, could be that matters? However, at the same time, I do not have any issues with other pods or with master branch of SQLite.swift

**Comment by jberkel** [comment]
@t4ec hmm, no that shouldn't matter. Maybe try removing the entire cache:  (`rm -rf ~/Library/Caches/CocoaPods/`)

Also, can you run `pod install --verbose` and post the output?

**Comment by t4ec** [comment]
@jberkel 

> Maybe try removing the entire cache: (rm -rf ~/Library/Caches/CocoaPods/)

Didn't help

> Also, can you run pod install --verbose and post the output?

Sure:

```
Fetching external sources
-> Pre-downloading: `SQLite.swift` from `https://github.com/stephencelis/SQLite.swift.git`, branch `swift-4`
  $ /usr/bin/git ls-remote https://github.com/stephencelis/SQLite.swift.git swift-4
  7312423d83bb713591a0d827a89fe664eba14451	refs/heads/swift-4
 > Git download
 > Git download
     $ /usr/bin/git clone https://github.com/stephencelis/SQLite.swift.git
     /var/folders/x7/h9kf68x52snfn4f6lr33snnm0000gn/T/d20170929-14211-6yxivj --template=
     Cloning into '/var/folders/x7/h9kf68x52snfn4f6lr33snnm0000gn/T/d20170929-14211-6yxivj'...
   $ /usr/bin/git -C /var/folders/x7/h9kf68x52snfn4f6lr33snnm0000gn/T/d20170929-14211-6yxivj checkout --quiet
   7312423d83bb713591a0d827a89fe664eba14451
[!] Unable to find a specification for 'SQLite.swift'.

/Users/machine/.rvm/gems/ruby-2.2.1/gems/cocoapods-1.2.0/lib/cocoapods/external_sources/abstract_external_source.rb:117:in `block in pre_download'
/Users/machine/.rvm/gems/ruby-2.2.1/gems/cocoapods-1.2.0/lib/cocoapods/user_interface.rb:85:in `titled_section'
/Users/machine/.rvm/gems/ruby-2.2.1/gems/cocoapods-1.2.0/lib/cocoapods/external_sources/abstract_external_source.rb:112:in `pre_download'
/Users/machine/.rvm/gems/ruby-2.2.1/gems/cocoapods-1.2.0/lib/cocoapods/external_sources/downloader_source.rb:13:in `fetch'
/Users/machine/.rvm/gems/ruby-2.2.1/gems/cocoapods-1.2.0/lib/cocoapods/installer/analyzer.rb:635:in `fetch_external_source'
/Users/machine/.rvm/gems/ruby-2.2.1/gems/cocoapods-1.2.0/lib/cocoapods/installer/analyzer.rb:611:in `block (2 levels) in fetch_external_sources'
/Users/machine/.rvm/gems/ruby-2.2.1/gems/cocoapods-1.2.0/lib/cocoapods/installer/analyzer.rb:610:in `each'
/Users/machine/.rvm/gems/ruby-2.2.1/gems/cocoapods-1.2.0/lib/cocoapods/installer/analyzer.rb:610:in `block in fetch_external_sources'
/Users/machine/.rvm/gems/ruby-2.2.1/gems/cocoapods-1.2.0/lib/cocoapods/user_interface.rb:64:in `section'
/Users/machine/.rvm/gems/ruby-2.2.1/gems/cocoapods-1.2.0/lib/cocoapods/installer/analyzer.rb:609:in `fetch_external_sources'
/Users/machine/.rvm/gems/ruby-2.2.1/gems/cocoapods-1.2.0/lib/cocoapods/installer/analyzer.rb:73:in `analyze'
/Users/machine/.rvm/gems/ruby-2.2.1/gems/cocoapods-1.2.0/lib/cocoapods/installer.rb:236:in `analyze'
/Users/machine/.rvm/gems/ruby-2.2.1/gems/cocoapods-1.2.0/lib/cocoapods/installer.rb:150:in `block in resolve_dependencies'
/Users/machine/.rvm/gems/ruby-2.2.1/gems/cocoapods-1.2.0/lib/cocoapods/user_interface.rb:64:in `section'
/Users/machine/.rvm/gems/ruby-2.2.1/gems/cocoapods-1.2.0/lib/cocoapods/installer.rb:149:in `resolve_dependencies'
/Users/machine/.rvm/gems/ruby-2.2.1/gems/cocoapods-1.2.0/lib/cocoapods/installer.rb:110:in `install!'
/Users/machine/.rvm/gems/ruby-2.2.1/gems/cocoapods-1.2.0/lib/cocoapods/command/install.rb:37:in `run'
/Users/machine/.rvm/gems/ruby-2.2.1@global/gems/claide-1.0.1/lib/claide/command.rb:334:in `run'
/Users/machine/.rvm/gems/ruby-2.2.1/gems/cocoapods-1.2.0/lib/cocoapods/command.rb:52:in `run'
/Users/machine/.rvm/gems/ruby-2.2.1/gems/cocoapods-1.2.0/bin/pod:55:in `<top (required)>'
/Users/machine/.rvm/gems/ruby-2.2.1/bin/pod:23:in `load'
/Users/machine/.rvm/gems/ruby-2.2.1/bin/pod:23:in `<main>'
/Users/machine/.rvm/gems/ruby-2.2.1/bin/ruby_executable_hooks:15:in `eval'
/Users/machine/.rvm/gems/ruby-2.2.1/bin/ruby_executable_hooks:15:in `<main>'

[!] Unable to load a podspec from `SQLite.swift.podspec`, skipping:

Pod::DSLError
```

**Comment by jberkel** [comment]
OK, managed to reproduce the error with cocoapods 1.2.0. Try upgrading to 1.3.1.

**Comment by t4ec** [comment]
@jberkel 

> OK, managed to reproduce the error with cocoapods 1.2.0. Try upgrading to 1.3.1.

Thanks! The issue was resolved after I had updated cocoapods to 1.3.1.
